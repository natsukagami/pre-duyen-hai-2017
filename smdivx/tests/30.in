95 130939728302213
