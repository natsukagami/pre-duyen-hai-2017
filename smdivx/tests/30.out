10000000000000000000000000000000000000000000000000000000000000000000000000000000059092560890521
